### PR TITLE
Add orthophoto rendering support. 

### PR DESCRIPTION
**Pull request by LeaFendd**
#2602 
You can use it like a perspective camera. Focal length is meaningless for orthophoto, so here fx&fy determines the imaging areas, the bigger fx/fy, the bigger imaging areas.

### CODE REVIEW

File: tests/cameras/test_cameras.py [hunk 1]
```diff
@@ -120,6 +120,18 @@ def test_pinhole_camera():
...
+def test_orthophoto_camera():
+    """Test that the orthophoto camera model works."""
+
+    c2w = torch.eye(4)[None, :3, :]
+    orthophoto_camera = Cameras(
+        cx=200.0, cy=400.0, fx=1.0, fy=1.0, camera_to_worlds=c2w, camera_type=CameraType.ORTHOPHOTO
+    )
+    camera_ray_bundle = orthophoto_camera.generate_rays(camera_indices=0)
+    assert isinstance(camera_ray_bundle, RayBundle)
+    assert camera_ray_bundle.shape == torch.Size([800, 400])
```
Comment (akristoffersen) [inline tests/cameras/test_cameras.py:132]: Could you add to this test to guarantee that the ray origins are correct, especially after rotation?

**Review by akristoffersen — CHANGES_REQUESTED** [review]
LGTM, I'd just flush out the test a bit.

File: tests/cameras/test_cameras.py (hunk 1 above)
Comment (LeaFendd) [reply tests/cameras/test_cameras.py:132]: Oh, I discovered a bug related to rotation issues caused by incorrectly using a left-handed coordinate system. The special case I used during testing (rotating 90 degrees along the z-axis) led me to overlook this issue. Thank you for your review. I have fixed this problem in the latest commit.
output
rotated

Comment (LeaFendd) [reply tests/cameras/test_cameras.py:132]: Sounds great, I will add a more complete test case, and make it work for _generate_rays_from_coords.

**Comment by akristoffersen** [comment]
LGTM, thanks for doing this :D

**Review by akristoffersen — APPROVED** [review]

**Comment by akristoffersen** [comment]
@LeaFendd Sorry, accidentally clicked the "Update" button on this PR, feel free to merge to main when you feel its ready.

**Comment by LeaFendd** [comment]
@akristoffersen I believe it's ready to be merged, and I'm happy to contribute to nerfstudio :D
Thanks for reviewing!

**Pull request merged**